stack-nightly.yaml

```diff
@@ -0,0 +1,12 @@
+resolver: nightly-2019-10-31
+
+packages:
+  - summoner-cli/
+  - summoner-tui/
+
+extra-deps:
+  - base-noprelude-4.13.0.0
```

Comment:
One more reason to use `mixins` approach with default prelude: `base-noprelude` is not going to be on Stackage unless we add it :disappointed: